From: Caleb Connolly <caleb.connolly@linaro.org>
Date: Tue, 27 Jun 2023 19:03:42 +0100
Subject: sdm845.config: 6.4 additions

ZRAM stuff From Alex, to be squashed on the next rebase
---
 arch/arm64/configs/sdm845.config | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm64/configs/sdm845.config b/arch/arm64/configs/sdm845.config
index 8501982..34b6134 100644
--- a/arch/arm64/configs/sdm845.config
+++ b/arch/arm64/configs/sdm845.config
@@ -106,6 +106,7 @@ CONFIG_UDMABUF=y
 CONFIG_DMABUF_HEAPS_CMA=y
 CONFIG_DMABUF_HEAPS_SYSTEM=y
 CONFIG_HZ_1000=y
+CONFIG_MODULE_COMPRESS_ZSTD=y
 
 # Needed for mounting userdata on android
 CONFIG_QFMT_V2=y
@@ -207,6 +208,7 @@ CONFIG_STACKTRACE=y
 CONFIG_VT=y
 CONFIG_CRYPTO_XTS=y
 CONFIG_TMPFS_XATTR=y
+CONFIG_BLK_DEV_DM=y
 CONFIG_DM_CRYPT=y
 CONFIG_BINFMT_MISC=m
 
@@ -249,7 +251,10 @@ CONFIG_IPVLAN=m
 CONFIG_ZSMALLOC=m
 CONFIG_ZSMALLOC_STAT=y
 CONFIG_ZRAM=m
+CONFIG_ZRAM_DEF_COMP_ZSTD=y
+CONFIG_ZRAM_DEF_COMP="zstd"
 CONFIG_ZRAM_MEMORY_TRACKING=y
+CONFIG_ZRAM_MULTI_COMP=y
 CONFIG_CRYPTO_LZ4=m
 CONFIG_LZ4_COMPRESS=m
 CONFIG_CRYPTO_LZO=m
